src/interfaces/plug_wallet.ts

```diff
@@ -37,18 +38,19 @@ export interface JSONWallet {
     icon?: string;
     connectedApps: Array<ConnectedApp>;
     assets: Assets;
-    nftCollections?: Array<{
+    collections?: Array<{
```

Comment:
I think you are missing the upgrade script for this change so that previous collections saved in storage are not lost. Same goes for custom collections as it should be initialized as an empty array (I think)